packages/shared/src/hooks/notifications/useBookmarkReminderEnrollment.ts

```diff
@@ -0,0 +1,15 @@
+import { useJustBookmarked } from '../bookmark';
+import { useConditionalFeature } from '../useConditionalFeature';
+import { feature } from '../../lib/featureManagement';
+import { Post } from '../../graphql/posts';
+
+export const useBookmarkReminderEnrollment = (post: Post): boolean => {
```

Comment:
Oeh nice, love this idea also easier cleanup I guess 😅 (as we'll always need the justBookmarked part)